### PR TITLE
Add a build config for fastboot

### PR DESCRIPTION
**Pull request by gdong1**
With this flag, platform could update the settings to
get the expected boot performance data

Signed-off-by: Guo Dong <guo.dong@intel.com>

### CODE REVIEW

File: BuildLoader.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def __init__(self, *args, **kwargs):
         self.HAVE_ACPI_TABLE       = 1
         self.HAVE_PSD_TABLE        = 0
         self.HAVE_SEED_LIST        = 0
+        self.HAVE_FAST_BOOT        = 0
```
Comment (mauricema) [inline BuildLoader.py:195]: Use  "self.ENABLE_FAST_BOOT" instead ?

File: BootloaderCorePkg/BootloaderCorePkg.dec [hunk 2]
```diff
@@ -133,7 +133,8 @@
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesSize       |     0x1000 | UINT16 | 0x200000E3
 
   # Size of the Hash store allocated in bootloader
-  gPlatformModuleTokenSpaceGuid.PcdHashStoreSize       | 0x00000200 | UINT32 | 0x200000F1
+  gPlatformModuleTokenSpaceGuid.PcdHashStoreSize          | 0x00000200 | UINT32 | 0x200000F1
+  gPlatformModuleTokenSpaceGuid.PcdFastBootEnabled        |  FALSE     | BOOLEAN| 0x200000F2
```
Comment (mauricema) [inline BootloaderCorePkg/BootloaderCorePkg.dec:137]: Add a comment on the expected behavior when this PCD is enabled

Comment (gdong1) [reply BootloaderCorePkg/BootloaderCorePkg.dec:137]: ok

File: BuildLoader.py (hunk 1 above)
Comment (gdong1) [reply BuildLoader.py:195]: sure

**Review by mauricema — APPROVED** [review]
Looks good to me.

File: BootloaderCorePkg/BootloaderCorePkg.dec (hunk 2 above)
Comment (myguitar) [reply BootloaderCorePkg/BootloaderCorePkg.dec:140]: Can we have a better name to avoid another "fastboot" term?

Comment (gdong1) [reply BootloaderCorePkg/BootloaderCorePkg.dec:140]: sorry, I had merged the patch for platform patch update. do you mean "MrcFastBoot" from FSP UPD?

**Pull request merged**